skbio/io/format/tests/data/blast6_values_under_wrong_column

```diff
@@ -0,0 +1 @@
+query1	8	75.00	subject2	2	0	1	8	2	0.060	9	11.5
```

Comment:
This file is still tracked, you'll need to remove it with `git rm`, commit, and push.